src/main/java/io/supertokens/session/info/SessionInformationHolder.java

```diff
@@ -44,4 +49,11 @@ public SessionInformationHolder(@Nonnull SessionInfo session, @Nullable TokenInf
         this.idRefreshToken = idRefreshToken;
         this.antiCsrfToken = antiCsrfToken;
     }
+
+    public JsonObject toJsonObject() {
+        JsonObject json = new Gson().toJsonTree(this).getAsJsonObject();
+        json.add("session", Utils.toJsonTreeWithNulls(session));
```

Comment:
are you sure this replaces the existing "session" key in the object and is not ignored?